lib/feature/question/presentation/pages/edit.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+import 'package:job_interview_practice/feature/question/presentation/widgets/appBar.dart';
+
+class EditPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: MyAppBar(),
+      body: Container(
+        padding: EdgeInsets.all(32.0),
+        child: Center(
+          child: Column(
+            children: <Widget>[
+              Text('Edit'),
+              RaisedButton(
+                onPressed: () => Navigator.of(context).pop(),
```

Comment:
TIP: if you have got equal headers of methods you can push it directly like here:
`onPressed: Navigator.of(context).pop`

Reply:
it looks very fancy IMHO 😄

Reply:
i didn't know that!!
In the future, I'll do that way.